packages/web3/src/exchange/exchange.ts

```diff
@@ -62,6 +62,7 @@ function getInputAddresses(tx: Transaction): Address[] {
   const inputAddresses: Address[] = []
   for (const input of tx.unsigned.inputs) {
     try {
+      if (input.unlockScript === '03') continue // SameAsPrevious
```

Comment:
IIRC, we have a constant somewhere for `03`, should reuse that one.

Reply:
We didn't define encoded `SameAsPrevious` before, so I added a constant for it in the codec